test/recipes/controls/aws_parallelcluster_config/network_interfaces_spec.rb

```diff
@@ -0,0 +1,67 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'network_interfaces_configured' do
```

Comment:
Nitpick. I would split this control in:
- one that checks `*.sh` scripts have been written correctly
- the other checking actual network interfaces configuration

Reply:
Done